src/view/ChewingEditor.cpp

```diff
@@ -178,6 +178,27 @@ void ChewingEditor::showAbout()
     aboutBox.exec();
 }
 
+void ChewingEditor::showDeleteConfirmWindow()
+{
+    QString text = QString("Do you want to delete this phrase?");
+
+    QMessageBox deleteBox(this);
+    deleteBox.setWindowTitle(tr("Delete phrase"));
+    deleteBox.setText(text);
+    deleteBox.setStandardButtons(QMessageBox::Yes | QMessageBox::No);
+    deleteBox.setDefaultButton(QMessageBox::No);
+
+    connect(
+        &deleteBox, SIGNAL(accepted()),
```

Comment:
connect the signal from the `QMessageBox::Yes` button to `remove()`
`deleteBox.button(QMessageBox::Yes), SIGNAL(clicked()),`